packages/google_maps_flutter/google_maps_flutter_web/lib/src/third_party/to_screen_location/to_screen_location.dart

```diff
@@ -28,30 +28,31 @@ import 'package:google_maps/google_maps.dart' as gmaps;
 /// (not of the whole screen/app).
 ///
 /// See: https://developers.google.com/maps/documentation/android-sdk/reference/com/google/android/libraries/maps/Projection#public-point-toscreenlocation-latlng-location
-gmaps.Point toScreenLocation(gmaps.GMap map, gmaps.LatLng coords) {
-  final num? zoom = map.zoom;
+gmaps.Point toScreenLocation(gmaps.Map map, gmaps.LatLng coords) {
   final gmaps.LatLngBounds? bounds = map.bounds;
   final gmaps.Projection? projection = map.projection;
```

Comment:
These could be moved below the assertions, so we can just do `final gmaps.Blah foo = map.foo!`, so we don't need other `!` below.